index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='UTF-8'/>
+    <meta name="description"/>
+    <title>Sedona cosik</title>
+  </head>
+
+  <body>
+    <div class="wrapper"><!-- main page -->
+      <header class="page-header">
+        <nav class="main-nav">
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
+            <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+            <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+            <li class="main-nav__item">HTML Academy</li>
+          </ul>
+          <button class="main-nav__toggle main-nav__toggle--close"></button>
+          <a class="main-nav__link" href="index.html"><img src="" class="main-nav__logo" alt="Sedona logo"></a>
+        </nav>
+
+        <div calss="intro">
+          <ul class="intro-list">
+            <li class="intro-list__item intro-list__item--top">Welcome to the gorgeous</li>
+            <li class="intro-list__item intro-list__item--middle">Sedona</li>
+            <li class="intro-list__item intro-list__item--bottom">"Because the Grand Canyon sucks!"</li>
+          </ul>
+        </div>
+      </header>
+
+      <section class="preamble">
+        <div class="preamble__inner">
+          <h2 class="preamble__header">SEDONA - a small town in the ARIZONA, deserves more!</h2>
+          <p class="preamble__paragraph">Consider 5 reasons why Sedona is cooler than Grand Canyon!</p>
+        </div>
+      </section>
+
+      <section class="place-one place-one--blue">
```

Comment:
Was it me who misslead you for this one? it was good before, it's one block. you don't need `place-one`, `-two`, etc. it should be just one block, and reuse it.